### PR TITLE
Feat/list

### PR DESCRIPTION
**Pull request by AmirBandarii**

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -31,6 +32,31 @@ server.get('/', (req, res) => {
...
+    res.render('list', { result });
+  } catch (error) {
+    return res.status(500).json({error: errors.internalServer});
+  }
+});
+
+server.get(`/list-id/:id`, async (req, res) => {
+  try{
+    const id = Number(req.params.id)
+    const getOneCustomerInfo =  await customerInfoService.getCustomerInfoById(id)
```
Comment (harutyunyan611) [inline src/server.ts]: variable name

**Review by harutyunyan611 — CHANGES_REQUESTED** [review]
Good job!

File: src/server.ts (hunk 1 above)
Comment (AmirBandarii) [reply src/server.ts]: done

**Review by harutyunyan611 — APPROVED** [review]

File: src/server.ts [hunk 2]
```diff
@@ -31,6 +33,45 @@ server.get('/', (req, res) => {
...
+    res.render('list', { result });
+  } catch (error) {
+    return res.status(500).json({error: errors.internalServer});
+  }
+});
+
+server.get(`/list-id/:id`, async (req, res) => {
+  try{
+    const customerInfoId = Number(req.params.id)
+    const getOneCustomerInfo =  await customerInfoService.getCustomerInfoById(customerInfoId)
```
Comment (harutyunyan611) [inline src/server.ts]: What is done I still see old variable name?

**Review by harutyunyan611 — CHANGES_REQUESTED** [review]

File: src/server.ts (hunk 2 above)
Comment (harutyunyan611) [reply src/server.ts]: `getOneCustomerInfo` is not vaid variable name

**Pull request merged**